zerver/migrations/0219_alter_field_realm_video_chat_provider.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-05-09 06:54
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+def update_existing_video_chat_provider_values(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Realm = apps.get_model('zerver', 'Realm')
+    all_realms = Realm.objects.all()
+
+    for realm in all_realms:
+        realm.video_chat_provider = Realm.VIDEO_CHAT_PROVIDERS.index(Realm.video_chat_provider)
+        realm.save(update_fields=["video_chat_provider"])
```

Comment:
So, the tests are failing because `test_migrations.py` doesn't really handle future migrations happening to the same model.  Generally, the solution is to delete the existing test in `test_migrations.py`.  

I think probably we want to switch to the strategy below:

```
diff --git a/zerver/tests/test_migrations.py b/zerver/tests/test_migrations.py
index 0b8057cd9b..36cb281e04 100644
--- a/zerver/tests/test_migrations.py
+++ b/zerver/tests/test_migrations.py
@@ -22,8 +22,12 @@ from zerver.models import get_stream
 #
 #   django.db.utils.OperationalError: cannot ALTER TABLE
 #   "zerver_subscription" because it has pending trigger events
+#
+# As a result, we generally mark these tests as skipped once they have
+# been tested for a migration being merged.
 
 class SubsNotificationSettingsTestCase(MigrationsTestCase):
+    __unittest_skip__ = True
 
     migrate_from = '0220_subscription_notification_settings'
     migrate_to = '0221_subscription_notifications_data_migration'
```

Also, you should rebase, since the latest migration number went up.

Reply:
I implemented your suggestions, and it is failing on tests coverage `./toos/test-backend --coverage`

Reply:
I'll debug this and merge.